Sources/LiveKit/protocols/RTCEngineDelegate.swift

```diff
@@ -18,4 +18,5 @@ protocol RTCEngineDelegate: AnyObject {
     func didDisconnect()
     func didFailToConnect(error: Error)
     func didReceive(packet: Livekit_UserPacket, kind: Livekit_DataPacket.Kind)
+    func didRemoteMuteChange(trackSid: String, muted: Bool)
```

Comment:
```suggestion
    func remoteMuteDidChange(trackSid: String, muted: Bool)
```

Reply:
Yea I aligned it with the other 'did's above haha but I'll change 😁